CellProfiler64.iss

```diff
@@ -25,11 +25,11 @@ AppSupportURL=http://www.cellprofiler.org
 AppUpdatesURL=http://www.cellprofiler.org
 DefaultDirName={pf64}\CellProfiler
```

Comment:
On Wed, Dec 9, 2015 at 1:40 PM, Allen Goodman <notifications@github.com>
wrote:

> > DefaultDirName={pf64}\CellProfiler
>
> Is this C:\Program Files\CellProfiler?
>
According to this (http://www.jrsoftware.org/ishelp/index.php?topic=consts),
yes.